test/archethic_web/api/jsonrpc/schemas/transaction_test.exs

```diff
@@ -131,7 +131,7 @@ defmodule ArchethicWeb.API.JsonRPC.TransactionSchemaTest do
 
       assert {:error,
               %{
-                "#/data/code" => "Expected value to have a maximum length of 24576 but was 49154."
+                "#/data/code" => "Expected value to have a maximum length of 24576 but was " <> _
```

Comment:
This test could be flaky.
Maybe you should ensure the compressed code is over the max size before calling the validation function